applications/launchpad/gui-react/src/components/TitleBar/index.tsx

```diff
@@ -80,6 +81,14 @@ const TitleBar = ({
     }
   }
 
+  const getSettingsIconColor = () => {
+    if (expertView !== 'hidden') {
+      return colors.light.textSecondary
```

Comment:
Not sure if we would like to extract this directly from the `colors`. I would suggest to even name directly this style in the themes, like: `titleBarIcon` (+ one to the `inverted` section).

Also, maybe we could make small improvement here like this:

```
const settingsIconColor =  useMemo(() => {
    if (expertView !== 'hidden') {
      return colors.light.textSecondary
    } else {
      return theme.helpTipText
    }
  }, [expertView])
```

Reply:
Changed to `theme.textSecondary` as it already exists.

`useMemo` causes issues here, breaks the icon colour when `expertView !== 'hidden'. Possibly not re-rendering when required?